### ISSUE TITLE
Lower camel case fields when using JSON

### ISSUE THREAD

**Issue by MungoRae** [joelittlejohn/jsonschema2pojo#563]
Is it currently possible to have fields generated from JSON be lower camel case?

For example if I have JSON:
```
{
  "Property1" : "Any Value",
  "PropertyTwo" : "any value2",
  "property3" : "Value"
}
```

I want the class to look like
```
public class Clazz {
    private String property1;
    private String propertyTwo;
    private String property3;
}
```
At the moment it creates the fields as `Property1` which goes against common code style.

Thanks

**Comment by joelittlejohn** [comment]
This is a regression, we should be generating lower camel case. Will definitely look to fix this. Thanks for raising it!

**Comment by joelittlejohn** [comment]
Closed by #565
